docs/content/any/getting-started/_index.md

```diff
@@ -8,3 +8,16 @@ any: false
 # Jump in
 
 Learn the basics of using Oso and get Oso set up in your application.
+
+{{% ifLang "python" %}}
+{{< callout "Early access to the next version of roles" "blue" >}}
+
+We just released early access to the next version of built-in support
+for Roles. Initially supporting SQLAlchemy, this library, includes
+improved support for common authorization models, like role based access
+control.
+
+[Check it out here!](/new-roles)
```

Comment:
@dhatch @gneray how about something like this

```suggestion
{{< callout "Built-in roles early access" "blue" >}}

We're hard at work on a major revision to Oso's built-in roles support.
We released [an early access version of our SQLAlchemy library](/new-roles)
containing the new features, which make it easy to model role-based access
control (RBAC) in Polar and enforce the policy over SQLAlchemy models.

[Check it out here!](/new-roles)
```

Reply:
We just discussed live a plan to:

1. Leave the SQLAlchemy callouts when the user has Python selected similar to as is (or maybe your suggestion)
2. Add callouts on roles pages for when users are not using Python that link to a higher level page describing features of the new roles work, with a link to try out in other languages that will email us.